understanding/non-text-contrast.html

```diff
@@ -108,395 +102,279 @@ <h1><span class="standalone-resource__type-of-guidance">Understanding
-                  technology.
+               <p>低いコントラストのコントロールは知覚するのが難しく、かつ視覚障害のある人に完全に見逃される可能性がある。同様に、ウェブページのコンテンツ又は機能を理解するためにグラフィックが必要とされる場合、コントラストを強化する支援技術の必要なくロービジョン又は他の障害のある人に知覚されるべきある。
                </p>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      				
-                     <p>The 3:1 contrast ratios referenced in this Success Criterion is intended to be treated
-                        as threshold values. When comparing the computed contrast ratio to the Success Criterion
-                        ratio, the computed values should not be rounded (e.g. 2.999:1 would not meet the
-                        3:1 threshold).
+                     <p>この達成基準で言及されている 3:1 のコントラスト比は、閾値として扱われることを意図している。算出されたコントラスト比を達成基準のコントラスト比と比較する場合、算出された値は丸めるべきではない（例えば、 2.999:1 は 3:1 という閾値を満たさない）。
                      </p>
                      			
                   </div>
                </div>
                <div class="note">
-                  <p class="note-title marker">Note</p>
+                  <p class="note-title marker">注記</p>
                   <div>
                      				
-                     <p>Because authors do not have control over user settings for font smoothing and anti-aliasing,
-                        when evaluating this
-                        					 Success Criterion, refer to the colors obtained from the user agent, or the
-                        underlying
-                        					 markup and stylesheets, rather than the non-text elements as presented on screen.
+                     <p>フォントスムージング及びアンチエイリアシングのユーザー設定をコンテンツ制作者はコントロールできないので、この達成基準を評価するときは、画面に表示される非テキスト要素よりも、ユーザエージェントで取得できる、つまり基礎となるマークアップ及びスタイルシートから得られる、色を参照すること。
                      </p>
                      				
-                     <p>Due to anti-aliasing, particularly thin lines and shapes of non-text elements may
-                        be rendered by user agents with
-                        					 a much fainter color than the actual color defined in the underlying CSS. This
-                        can lead to situations where
-                        					 non-text elements have a contrast ratio that nominally passes the Success Criterion,
-                        but have a much lower contrast
-                        					 in practice. In these cases, best practice would be for authors to avoid particularly
-                        thin lines and shapes,
-                        					 or to use a combination of colors that exceeds the normative requirements of
-                        this Success Criterion.
-                        				
+                     <p>アンチエイリアスが原因で、非テキスト要素における特に細い線及び図形は、ユーザエージェントによって、基礎となる CSS で定義された実際の色よりもずっと淡い色で表示されることがある。このため、非テキスト要素のコントラスト比が一応は達成基準を満たしていても、実際にはコントラストがかなり低いという状況が起こり得る。このような場合、ベストプラクティスは、コンテンツ制作者が特に細い線及び図形を避けるか、この達成基準の規範的要件を上回る色の組み合わせを用いることであろう。
```

Comment:
こちらもSC 1.4.3のコメントに寄せてみました。
```suggestion
                     <p>アンチエイリシングが原因で、非テキスト要素の特に細い線及び図形は、ユーザエージェントによって、基礎となる CSS で定義された実際の色よりもずっとうすい色でレンダリングされることがある。このため、非テキスト要素のコントラスト比が名目上は達成基準を満たしていても、実際にはコントラストがかなり低いという状況が起こり得る。このような場合、ベストプラクティスは、コンテンツ制作者が特に細い線及び図形を避けるか、この達成基準の規範的要件を上回る色の組み合わせを用いることである。
```